<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>529</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>思源黑体</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="cursor">
   <cursorShape>IBeamCursor</cursorShape>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>牛逼nc文件读取</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="click_print">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>40</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>思源黑体</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>选择文件</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="filename_line">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>40</y>
      <width>401</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>思源黑体</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string>请读取文件</string>
    </property>
   </widget>
   <widget class="QWidget" name="plot" native="true">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>90</y>
      <width>621</width>
      <height>381</height>
     </rect>
    </property>
    <widget class="QTextBrowser" name="fileProperties">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>621</width>
       <height>381</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>思源黑体</family>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
   </widget>
   <widget class="QComboBox" name="vars_box">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="currentText">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>思源黑体</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>变量</string>
    </property>
   </widget>
   <widget class="QPushButton" name="draw">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>思源黑体</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>可视化</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Prev_plot">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>390</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>思源黑体</family>
      <pointsize>10</pointsize>
      <italic>false</italic>
      <bold>false</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="text">
     <string>上一张</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Next_plot">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>390</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>思源黑体</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>下一张</string>
    </property>
   </widget>
   <widget class="QPushButton" name="close_plot">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>430</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>思源黑体</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>关闭图像</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>780</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
